packages/foreman/katello-certs-tools/katello-certs-tools.spec

```diff
@@ -42,10 +36,8 @@ pathfix.py -pni "%{__python3} %{py3_shbang_opts}" . katello-ssl-tool
 %install
 %if 0%{?rhel} == 7
 %{__python2} setup.py install --skip-build --root $RPM_BUILD_ROOT
```

Comment:
Can this now use `%{py2_install}`?

Reply:
Why couldn't it before? I have no idea :)